src/connection.cpp

```diff
@@ -766,6 +782,19 @@ namespace oxen::quic
         // config values
         params.initial_max_streams_bidi = (uconfig.max_streams) ? uconfig.max_streams : DEFAULT_MAX_BIDI_STREAMS;
 
+        if (uconfig.datagram_support)
+        {
+            params.max_datagram_frame_size = 65535;
```

Comment:
Deserves a comment along the lines of:
```
// This is effectively an "unlimited" value, which lets us accept any size that fits into a QUIC packet.
```
because otherwise I was confused by us not being able to accept anything remotely as large as this.